axelrod/graph.py

```diff
@@ -6,7 +6,11 @@
 """
 
 from collections import defaultdict
+from typing import List, Dict, Hashable, Tuple
+from numbers import Number
```

Comment:
We can assume that the weight is a float IMO rather than a Number

Reply:
Sure, will make changes ASAP.